Fixes to ZJSON format doc (#3838)
brimdata/zed: docs/formats/zjson.md

A ZJSON file is composed of ZJSON objects formatted as
[newline delimited JSON (NDJSON)](http://ndjson.org/).
e.g., the [zq](../commands/zq.md) CLI command
writes its ZJSON output as lines of NDJSON.

## 4. Example

Here is an example that illustrates values of a repeated type,

> nesting, records, array, and union. Starting with file `input.zson`:

Reviewer: ```suggestion
nesting, records, array, and union. Consider the file `input.zson`:
```
